cmd/evm/internal/t8ntool/execution.go

```diff
@@ -343,6 +343,7 @@ func (pre *Prestate) Apply(vmConfig vm.Config, chainConfig *params.ChainConfig,
 		// Amount is in gwei, turn into wei
 		amount := new(big.Int).Mul(new(big.Int).SetUint64(w.Amount), big.NewInt(params.GWei))
 		statedb.AddBalance(w.Address, amount)
+		statedb.Witness().TouchFullAccount(w.Address[:], true)
```

Comment:
In the execution code we did this correctly, but was missing in the filling.

Reply:
yeah it's really dumb that we are copying so much code